src/Workspaces/Core/Portable/AddImports/AddImportHelpers.cs

```diff
@@ -47,10 +47,13 @@ internal static class AddImportHelpers
                     // And move it to the new using.
                     var originalFirstUsingCurrentIndex = newImports.IndexOf(originalFirstUsing);
 
-                    var newFirstUsing = newImports[0].WithLeadingTrivia(originalFirstUsing.GetLeadingTrivia())
-                                                     .WithAppendedTrailingTrivia(syntaxFacts.ElasticCarriageReturnLineFeed);
+                    newImports[0] = newImports[0].WithLeadingTrivia(originalFirstUsing.GetLeadingTrivia());
+
+                    if (!syntaxFacts.IsEndOfLineTrivia(newImports[0].GetTrailingTrivia().LastOrDefault()))
```

Comment:
Looks like this line is causing the following error in signed build -
AddImports\AddImportHelpers.cs(52,56): error RS0014: Do not use Enumerable methods on indexable collections. Instead use the collection directly [E:\A\_work\24\s\src\Workspaces\Core\Portable\Workspaces.csproj]

I am not sure why this didn't break CI.

@jasonmalinowski @jaredpar @dotnet/roslyn-infrastructure Does this mean CI build is configured differently from signed build?